job-server/src/main/resources/application.conf

```diff
@@ -319,17 +319,33 @@ akka.http.server {
   parsing.max-content-length = 30m
 }
 
-shiro {
-    # activate authentication (and authorization)
-    authentication = off
+access-control {
+  # note that ssl-encryption should also be on when authentication/authorization is on to avoid that passwords
+  # can be sniffed out
+  # Authentication provider class. Implementations have to extend SJSAuthenticator
+  provider = spark.jobserver.auth.AllowAllAccessControl
+  # Time out for job server to wait for authentication requests
+  auth-timeout = 10 s
+  shiro {
     # absolute path to shiro config file, including file name
     config.path = "/some/path/shiro.ini"
-    # note that ssl-encryption should also be on when authentication is on to avoid that passwords
-    # can be sniffed out
-    # Time out for job server to wait for authentication requests
-    authentication-timeout = 10 s
     # when authentication is on, then this flag controls whether the authenticated user
     # is always used as the proxy-user when starting contexts
     # (note that this will overwrite the spark.proxy.user parameter that may be provided to the POST /contexts/new-context command)
     use-as-proxy-user = off
+  }
+
+  keycloak {
+    authServerUrl = "https://example.com/auth"
+    realmName = "realm"
+    client = "client"
+    clientSecret = "clientSecret"
+  }
+
+  akka.http.caching.lfu-cache {
```

Comment:
Are these settings automatically picked up even if they are inside `access-control` section?

Reply:
Yes, I only pass the `access-control` subsection to the cache settings creation.